src/applications/check-in/locales/en/translation.json

```diff
@@ -440,5 +440,7 @@
   "find-out-how-to-check-in-opens-in-new-tab": "Find out how to check in (opens in new tab)",
   "we-cant-show-all-information": "We can’t show all your information right now",
   "some-appointment-information-not-available": "Some of your appointment information is not available right now.",
-  "find-all-appointment-information": "To find all your appointment information, finish checking in for this appointment first. Then go to appointments on VA.gov."
+  "find-all-appointment-information": "To find all your appointment information, finish {{ type }} first. Then go to appointments on VA.gov.",
```

Comment:
I'm not sure but this could be difficult to translate. I would make it two separate keys.